test/component/home/navigation_home_test.dart

```diff
@@ -10,134 +10,140 @@ import "package:proxima/views/navigation/routes.dart";
-    expect(newPostPage, findsOneWidget);
-
-    //Go back to the home page
-    final leadingBackButton =
-        find.byKey(LeadingBackButton.leadingBackButtonKey);
-    expect(leadingBackButton, findsOneWidget);
-    await tester.tap(leadingBackButton);
-    await tester.pumpAndSettle();
-
-    //Check that the home page is displayed
-    expect(homePage, findsOneWidget);
+    nonEmptyMockedPage = ProviderScope(
+      overrides: [
+        postOverviewProvider.overrideWith(
+          () => MockHomeViewModel(
+            build: () async => testPosts,
+          ),
+        ),
+      ],
+      child: homePageApp,
+    );
   });
 
-  testWidgets(
-      "new post flow without post, using button creation text and come back to home page using back button",
-      (tester) async {
-    await tester.pumpWidget(emptyMockedPage);
-    await tester.pumpAndSettle();
-
-    // Check that the home page is displayed
-    final homePage = find.byType(HomePage);
-    expect(homePage, findsOneWidget);
-
-    //Click on the new post button on the home page
-    final newPostButtonText = find.byKey(PostFeed.newPostButtonTextKey);
-    expect(newPostButtonText, findsOneWidget);
-    await tester.tap(newPostButtonText);
-    await tester.pumpAndSettle();
-
-    //Check that the new post page is displayed
-    final newPostPage = find.byType(NewPostPage);
-    expect(newPostPage, findsOneWidget);
-
-    //Go back to the home page
-    final leadingBackButton =
-        find.byKey(LeadingBackButton.leadingBackButtonKey);
-    expect(leadingBackButton, findsOneWidget);
-    await tester.tap(leadingBackButton);
-    await tester.pumpAndSettle();
-
-    //Check that the home page is displayed
-    expect(homePage, findsOneWidget);
+  group("Post creation flow", () {
+    testWidgets("From non-empty feed, flow to create a post using bottomBar",
+        (tester) async {
+      await tester.pumpWidget(nonEmptyMockedPage);
+      await tester.pumpAndSettle();
+
+      // Check that the home page is displayed
+      final homePage = find.byType(HomePage);
+      expect(homePage, findsOneWidget);
+
+      //Click on the middle element of the bottombar
+      final bottomBar = find.byKey(NavigationBottomBar.navigationBottomBarKey);
+      await tester.tap(
+        find.descendant(
+          of: bottomBar,
+          matching: find
+              .byType(NavigationDestination)
+              .at(NavigationbarRoutes.addPost.index),
+        ),
+      );
+      await tester.pumpAndSettle();
+
+      //Check that the new post page is displayed
+      final newPostPage = find.byType(NewPostPage);
+      expect(newPostPage, findsOneWidget);
+
+      //Go back to the home page
+      await tester.tap(find.byKey(LeadingBackButton.leadingBackButtonKey));
+      await tester.pumpAndSettle();
+
+      //Check that the home page is displayed
+      expect(homePage, findsOneWidget);
+    });
+
+    testWidgets("From empty feed, flow to create a post using bottom bar",
```

Comment:
```suggestion
    testWidgets("From empty feed, flow to create a post using bottomBar",
```

Reply:
Will chose bottom bar, so that we refer to the type of component in general, instead of bottomBar that is more suited for a variable name.
Sorry @TheTexanCodeur x)